src/geometries/CapsuleGeometry.js

```diff
@@ -22,11 +22,25 @@ class CapsuleGeometry extends LatheGeometry {
 	 * @param {number} [length=1] - Length of the middle section.
 	 * @param {number} [capSegments=4] - Number of curve segments used to build the caps.
 	 * @param {number} [radialSegments=8] - Number of segmented faces around the circumference of the capsule.
+	 * @param {number} [heightSegments=1] - Number of rows of faces along the height of the capsule.
 	 */
-	constructor( radius = 1, length = 1, capSegments = 4, radialSegments = 8 ) {
+	constructor( radius = 1, length = 1, capSegments = 4, radialSegments = 8, heightSegments = 1 ) {
```

Comment:
Why `heightSegments`? Why not `lengthSegments`

Reply:
If anything I would think `length` should change to `height`, and that would be a useful change.

Looking at other geometries:
`BoxGeometry` -> `height`
`ConeGeometry` -> `height`
`CylinderGeometry` -> `height`
and so on, with `heightSegments`, etc. 

`height` would be more consistent within itself and other components. I'll make that change.

Reply:
I vote to stick to the `height` notation in this case.